curio/app/src/components/recorder.js

```diff
@@ -1,11 +1,34 @@
-
-import {Recorder} from 'react-voice-recorder'
+// import {Recorder} from 'react-voice-recorder'
```

Comment:
Ditto!

Reply:
Making sure that these points are resolved in the next commit.